### ISSUE TITLE
Investigate using OpenTomb for Tomb Raider 1-5

### ISSUE THREAD

**Issue by JoshuaFern** [luxtorpeda-dev/packages#78]
* Project link: https://github.com/opentomb/OpenTomb
* Project license: LGPLv3
* Does it work on 64-bit only Linux: Maybe
* Is source code available?: Yes
* Is source code stored in Git?: Yes
* List of Steam AppIds of supported games:
    - https://steamdb.info/app/224960/
    - https://steamdb.info/app/225300/
    - https://steamdb.info/app/225320/
    - https://steamdb.info/app/224980/
    - https://steamdb.info/app/225000/

## Additional information

I have not yet tested if OpenTomb works with Steam releases.

**Comment by d10sfan** [comment]
Have a built version somewhat working with this: https://github.com/luxtorpeda-dev/packages/pull/86

Remaining pieces:

- Figure out how to get the game assets in the right format. There's a GAME.GOG file that has to be extracted on run time. Script that may be a starting point (after the innoextract, that isn't needed), but bchunk isn't normally installed: https://github.com/cylgom/gog_tomb_raider_extractor/blob/master/extract.sh
- Convert or download music assets (OpenTomb has hosted converted assets), or provide ffmpeg?
- Download loading screens from opentomb linked location

**Comment by d10sfan** [comment]
@JoshuaFern 

I've implemented OpenTomb for 2-5. All of those looks like they just needed the data in the right area, so I've created setup scripts for each one. I tested 2, but not the rest.

For Tomb Raider I, I ended up using OpenLara, as it looks to have progressed further (has a full menu and the like), but for the other games it sounds like it's still very WIP. For this, I have built two utilities that are brought along with the openlara package, that allows for the GAME.GOG file to be extracted.

For OpenTomb, it requires using the console to switch levels, but that seems to basically work. I put a note in the packages comments about that limitation.

For all of them, music is being downloaded from OpenTomb hosted source (converted to OGG).
